wasm/module.go

```diff
@@ -78,3 +80,61 @@ func DecodeModule(binary []byte) (*Module, error) {
 	}
 	return ret, nil
 }
+
+func (m *Module) GetFunctionNames() (map[uint32]string, error) {
```

Comment:
this one screams "unit test me", especially with realistic inputs

Reply:
added!